src/main/webapp/resources/js/pages/analysis/components/AnalysisSteps.jsx

```diff
@@ -54,7 +54,7 @@ export function AnalysisSteps() {
       <Step
         title={getI18N("AnalysisSteps.completed")}
         icon={
-          analysisContext.analysisState === "COMPLETED" ? <Running /> : null
+          analysisContext.analysisState === "COMPLETED" ? <Success /> : null
```

Comment:
FYI, I think this can be simplified to:

```javascript
analysisContext.analysisState === "COMPLETED" && <Success />;
```

So you do not need to add the `null`